components/NavbarItem/NavbarItem.js

```diff
@@ -12,7 +12,7 @@ export function NavbarItem({ children, isActive }) {
   );
 }
 
-const NormalListItem = styled.li`
+const NormalListItem = styled.div`
   padding: 0.8rem;
```

Comment:
If you use rem your padding is getting bigger when user enlarges the font-size. Do you want that? If not, you could use 13px (not exactly the same, but only 0,2px difference on 16px 🙂 )